src/ralph/conf.py

```diff
@@ -210,5 +213,18 @@ def LOCALE_ENCODING(self) -> str:  # pylint: disable=invalid-name
         """Returns Ralph's default locale encoding."""
         return self._CORE.LOCALE_ENCODING
 
+    @root_validator(allow_reuse=True)
+    @classmethod
+    def check_restriction_compatibility(cls, values):
+        """Raise an error if scopes are being used without authority restriction."""
+        if values.get("LRS_RESTRICT_BY_SCOPES") and not values.get(
+            "LRS_RESTRICT_BY_AUTHORITY"
+        ):
+            raise ConfigurationException(
+                "`LRS_RESTRICT_BY_AUTHORITY` must be set to `True` if using "
+                "`LRS_RESTRICT_BY_SCOPES=True`"
```

Comment:
Not sure back-quotes are required for exception strings:

```suggestion
                "LRS_RESTRICT_BY_AUTHORITY must be set to True if using "
                "LRS_RESTRICT_BY_SCOPES=True"
```

Reply:
Removed them !